lib/RepairSteps/CurrentViewNameRepairStep.php

```diff
@@ -85,6 +85,14 @@ public function run(IOutput $output):void {
 					$this->config->setUserValue($userId, 'calendar', 'currentView', 'timeGridWeek');
 					break;
 
+				case 'listWeek':
```

Comment:
This file is only a migration step for users coming from Calendar 1.x.

Version 1 had no list view, so there is no need to change the saved view.
Please remove lines 88 to 95.

Reply:
Done.